apps/openmw/mwworld/worldimp.cpp

```diff
@@ -1535,6 +1535,11 @@ namespace MWWorld
         }
     }
 
+    osg::ref_ptr<osg::Node> World::getInstance (const std::string& modelName)
```

Comment:
World isn't supposed to expose rendering details. Can we remove that function and move whatever uses it inside the rendering system?